react/features/remote-video-menu/components/web/MuteEveryoneDialog.js

```diff
@@ -0,0 +1,137 @@
+/* @flow */
```

Comment:
Please use `// @flow` instead.

Reply:
thanks! changing. (fun fact: there are 104 instances of using `/* @flow */ in other places in the code :D)`